tests/gis_tests/geoapp/tests.py

```diff
@@ -532,6 +533,47 @@ def test_covers_lookup(self):
             qs = State.objects.filter(poly__covers=poly)
             self.assertSequenceEqual(qs, [state])
 
+    @skipUnlessDBFeature("supports_relate_lookup")
+    def test_relate_function(self):
+        within_mask = "T*F**F***"
+        ks = State.objects.get(name="Kansas")
+        qs = City.objects.annotate(
+            relate=Relate("point", ks.poly),
+            mask=Relate("point", ks.poly, within_mask),
+        )
+        self.assertQuerySetEqual(
+            list(qs),
+            (
+                ("Houston", "FF0FFF212", False),
+                ("Dallas", "FF0FFF212", False),
+                ("Oklahoma City", "FF0FFF212", False),
+                ("Wellington", "FF0FFF212", False),
+                ("Pueblo", "FF0FFF212", False),
+                ("Lawrence", "0FFFFF212", True),
```

Comment:
With GEOS 3.13 this becomes...
```suggestion
                ("Lawrence", "0FFFFF212", False),
```